lib/github_s.mli

```diff
@@ -339,10 +339,26 @@ module type Github = sig
         fires for responses with status [expected_code] and applies
         [parse]. *)
 
+    (** The [Media_type] module exposes a range of HTTP [Accept]
+        header values suitable for use in GitHub API requests *)
+    module Media_type : sig
+      val v3 : string
+      (** [v3] corresponds to [application/vnd.github.v3+json], the
+          media-type used by the
+          {{:https://developer.github.com/v3/} GitHub API v3} *)
+
+      val experimental : string
```

Comment:
I'm not sure this should be exposed in the interface as a lot of these media types are temporary.